### DISCUSSION TITLE
Widgets dissappearing in ctkinter

### DISCUSSION THREAD

**Discussion by shubhwins07** [Akascape/py-window-styles#27] [Q&A]
In my customtkinter programs(all of them), when I try to implement pywinstyles, the app does get transparent but the widgets just disappear, I have tried   .update()     .after()    but none of them work, when I capture the screen(both screenshot and recording using window's tools), then the widgets are somehow visible but when i capture screen with OBS studio it isn't visible as in normal use. Could you please help me? I have written the code below-

![Screenshot 2024-05-13 190055](https://github.com/Akascape/py-window-styles/assets/131468983/f71164ec-32b9-403f-b04b-9eccc6dd92d7)
**Snipping tool screenshot**

![image](https://github.com/Akascape/py-window-styles/assets/131468983/3c31ba71-2a99-4ac0-b7d2-0ed96f1efd19)
**An image from OBS recording**

```
import customtkinter as ctk
import time
import pywinstyles





window = ctk.CTk()
window.title("My Clock App")
window.geometry("300x200")

pywinstyles.apply_style(window, "acrylic")

stopwatchRunning = False
total_time = 0.0
start_time = None

def printtime(label):
    global total_time
    global start_time
    if stopwatchRunning:
        if start_time is None:
            # Stopwatch just started or resumed, record the current time
            start_time = time.time()
        else:
            # Stopwatch is still running, update the total time and the label
            currenttime = time.time()
            elapsed = currenttime - start_time
            total_time += elapsed
            total_time=round(total_time, 2)
            label.configure(text=f"{total_time:.2f} secs")
            # Record the current time for the next update
            start_time = currenttime
        label.after(50, printtime, label)

        
        
        
        


def start_stopwatch():
    global stopwatchRunning
    if not stopwatchRunning:
        stopwatchRunning = True
        printtime(label)
    
def stop_stopwatch():
    global stopwatchRunning
    global start_time
    stopwatchRunning = False
    # Reset the start time so the next start will know it's a new start
    start_time = None
    
def restart_stopwatch():
    global total_time
    global start_time
    global stopwatchRunning
    total_time = 0.0
    start_time = None
    stopwatchRunning = False
    label.configure(text="0.00 secs")
    





label = ctk.CTkLabel(window, text="00:00.00 secs", font=("SF Pro Display", 24))
start_button = ctk.CTkButton(window, text="Start",command=start_stopwatch)
stop_button = ctk.CTkButton(window, text="Stop", command=stop_stopwatch)
restart = ctk.CTkButton(window, text="Restart",command=restart_stopwatch)



label.pack(pady=10)
start_button.pack(pady=5)
stop_button.pack(pady=5)
restart.pack(pady=5)




window.mainloop()
```

**Comment by Akascape** [comment]
@shubhwins07 Try updating OBS studio, I tried both obs studio and snipping tool for recording and nothing like that is happening, also try changing the python env, 
![image](https://github.com/Akascape/py-window-styles/assets/89206401/d2b42723-6730-4b54-ba93-94d0207a5578)

**Comment by shubhwins07** [comment]
Created a new venv but still not working


https://github.com/Akascape/py-window-styles/assets/131468983/eff26153-c263-4317-9e1e-cc25dee3b374




Also tried this in Jetbrains Pycharm IDE still the same results


![image](https://github.com/Akascape/py-window-styles/assets/131468983/a4b9059d-3ea6-474a-9533-d98915b72a5c)

**Reply by Akascape** [reply]
@shubhwins07 
Did you update obs? 
which version are you using? 
Also try with other themes, is it working without pywinstyles?

**Comment by shubhwins07** [comment]
1. Now i have, will attach the Updated footage below
2. In VS code i am using python 3.12.1 and in Pycharm i used python 3.10 for demonstration.
3. If you are referring to themes set using `ctk.set_default_color_theme('theme.json')` method then yes it is working properly.



https://github.com/Akascape/py-window-styles/assets/131468983/912116d0-45b8-41f6-87e0-9c3d3e46775d
**This was recorded using OBS studio 30.1.2, the latest version**

**Comment by JiaPai12138** [comment]
Same issue, Python 3.10~3.13, x64
Windows 11 [Version 10.0.26100.1] LTSC
i am using snipaste to make it appear...
